src/inmanta/agent/agent.py

```diff
@@ -1168,6 +1172,8 @@ async def get_code(
                 )
                 resource_install_specs.append(resource_install_spec)
 
+                # TODO: cache is running ahead?
```

Comment:
Is this TODO meant to remain?

Reply:
I removed it, updated the comment to be more accurate, but I added another comment as this cache is a pre-existing memory leak.